src/wp-includes/widgets.php

```diff
@@ -1072,22 +1072,37 @@ function wp_get_sidebar( $id ) {
  * Set the sidebar widget option to update sidebars.
  *
  * @since 2.2.0
+ * @since 6.0.1 Added the optional $refresh_global_sidebars_widgets argument.
  * @access private
  *
  * @global array $_wp_sidebars_widgets
- * @param array $sidebars_widgets Sidebar widgets and their settings.
+ * @global array $sidebars_widgets
+ *
+ * @param array   $new_sidebars_widgets            Sidebar widgets and their settings.
+ * @param boolean $refresh_global_sidebars_widgets Optional. Whether to update $sidebars_widgets
```

Comment:
```suggestion
 * @param bool  $refresh_global_sidebars_widgets Optional. Whether to update $sidebars_widgets.
```
- Core uses the short type`bool`.
- Added ending period.